pages/api/auth/[...nextauth].ts

```diff
@@ -4,9 +4,18 @@ import GoogleProvider from "next-auth/providers/google"
 
 let googleClientId = process.env.GOOGLE_CLIENT_ID
 let googleClientSecret = process.env.GOOGLE_CLIENT_SECRET
+let baseUrl = process.env.FRONT_BASE_URL
```

Comment:
Do we need the env var for frontend? Can works with a const?

Reply:
For now that isn't necessary. In the future I'd like to redirect to game selection page after successful login.
At the moment we need that kind of variable, I think.
Sorry, I accidentally pushed that change.
@FilippoQuattrocchi